blog/2023-05-19-how-10101-app-protects-you/index.md

```diff
@@ -0,0 +1,113 @@
+But it's not only about the costs; when you go on chain by executing a non-collaborative settlement you have to wait longer until the funds arrive in your on-chain wallet.
+So, you try to avoid non-collaborative settlement - you really only want to use it in a worst case scenario.
+
+In order to collaboratively close or extend a position your 10101 app will have to be online because you have to sign one or multiple transactions with your trading partner.
+Since the setup is self-custodial and the keys never leave your phone the app has to be online for signing.
+If you open a perpetual future in 10101, and then you close the app, eventually the app will have to be online again to prolong or close the future.
+In case the app does not come online your trading party will eventually have to assume you are not responsive anymore and will have to trigger a non-collaborative settlement.
+
+The 10101 app, and 10101 as Lightning Service Provider (LSP) help you to avoid a non-collaborative settlement.
+10101 ensures that you receive push notifications that make it clear when an action is required.
+This enables you get informed and trigger actions to avoid an unnecessary non-collaborative settlement.
+
+![](./PositionCloseNotifications.png)
+
+These services are convenience features.
+If you ignore the notifications it just means that the execution may become more expensive because a non-collaborative settlement may be triggered by the trading counterparty.
+While push notifications will be sent out by 10101, it does not have to be 10101 that provides this feature.
+To make this setup more resistant against censorship all our code is open source and other service providers could eventually step in and provide these services.
+This could be handled similar to a watchtower setup in the Lightning network; watchtower providers could monitor position expiry maturity and notify the user as necessary.
+
+## Automation in a self-sovereign setup
+
+Arguably, it can be difficult to react to notifications on the phone immediately.
+You might be busy at your workplace and not be able to react when a notification arrives.
+
+Additionally, it can be hard to monitor the price all the time without automated order creation.
+For many traders the ability to create limit orders at a certain price is essential for their trading strategy.
+For market makers limit orders, and the ability for an automated trade setup is very important.
+
+During our beta phase we are exploring ways for a more convenient yet self-custodial setup for traders and market makers.
+
+### 10101 in your self-sovereign node setup
+
+One way to achieve self-sovereign automation is plugging the 10101 app into your own node setup.
+This could be for example a self-sovereign Lightning node setup on [Umbrel](https://umbrel.com/).
+You install the 10101 app on your Umbrel node but configure your trade settings in the 10101 app on your phone.
```

Comment:
We definitely should; best would be to define and runx experiments for it!
Interviews, polls, questionnaires can be a starting point.

---

I would start with the notifications and learn from them to take further decisions.
It will be interesting to see if notifications are actually a sufficient way to get users to act when they have to - and if users care about higher costs because e.g. a non-collab settlement was triggered because they did not react. We can also experiment with a "holding fee", where we don't trigger non-collab and give the user some time after expiry to come online and collab close.

For limit orders we have similar problems; and we will have to be careful to avoid a "free option" for our users by notifying them about a match and having a price guarantee for X time to execute the match.

The self-sovereign always-online raspberry node solution is great, but we will run into the same problem as with itchysats then: Users that want certain features would need certain technical skills to have the features because they require their own node. 
Maybe that's fine - maybe it's enough for the non-technical users to have market orders and to get notification to come online to close a position. 
The federation *could* be one way to create an always-online environment - but that would, obviously, not be fully self-custodial anymore.

It's most important for me that what the blogpost writes up is coherent :)